stdlib/3/io.pyi

```diff
@@ -53,6 +53,7 @@ class RawIOBase(IOBase):
     def read(self, size: int = ...) -> Optional[bytes]: ...
 
 class BufferedIOBase(IOBase):
+    raw: RawIOBase
```

Comment:
Thanks! Could you add a comment like "not part of the required BufferedIOBase API" or something similar per the documentation?

Reply:
Done